scripts/powershell/copy-docs-to-blobstorage.ps1

```diff
@@ -0,0 +1,167 @@
+# Note, due to how `Expand-Archive` is leveraged in this script, powershell
```

Comment:
We should figure out if this should go into eng/common or if we should break it up and put it into the language repo's and have only part of this be in eng/common.

Reply:
I created this [issue](https://github.com/Azure/azure-sdk-tools/issues/435) to track that.